{"uid":"596d96befb290197","name":"test_vul_low","fullName":"test_case.test_vul_distribution.test_vul_low.TestCase#test_vul_low","historyId":"eea309db7e207d17d0049434b71d655b","time":{"start":1753957654456,"stop":1753957654456,"duration":0},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_vul_distribution.test_vul_low.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页风险api分布饼图低危弱点api总数数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【api列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/assets/list'\n                payload = {\"vul_level_ids\": [2], \"page_num\": 1, \"page_size\": 10, \"time_layout\": \"2006-01-02 15:04:05\",\n                           \"sort\": -1}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_vul_distribution\\test_vul_low.py:33: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1753957651477,"stop":1753957651562,"duration":85},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"event_loop_policy","time":{"start":1753957651563,"stop":1753957651563,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"_xunit_setup_class_fixture_TestCase","time":{"start":1753957654456,"stop":1753957654469,"duration":13},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 792, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, cls)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 720, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_vul_distribution\\test_vul_low.py\", line 33, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[{"name":"【资产】-【api列表】接口","time":{"start":1753957654457,"stop":1753957654457,"duration":0},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_vul_distribution\\test_vul_low.py\", line 33, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":1,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_vul_distribution.test_vul_low.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页风险api分布饼图低危弱点api总数数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【api列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/assets/list'\n                payload = {\"vul_level_ids\": [2], \"page_num\": 1, \"page_size\": 10, \"time_layout\": \"2006-01-02 15:04:05\",\n                           \"sort\": -1}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_vul_distribution\\test_vul_low.py:33: Failed","steps":[],"attachments":[{"uid":"f8010a873b743736","name":"log","source":"f8010a873b743736.txt","type":"text/plain","size":228}],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":1},"afterStages":[],"labels":[{"name":"story","value":"【低危弱点api总数】"},{"name":"feature","value":"【首页】-【风险api分布饼图】-【低危弱点api总数】"},{"name":"severity","value":"normal"},{"name":"parentSuite","value":"test_case.test_vul_distribution"},{"name":"suite","value":"test_vul_low"},{"name":"subSuite","value":"TestCase"},{"name":"host","value":"5CD2323XZM"},{"name":"thread","value":"11380-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_case.test_vul_distribution.test_vul_low"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"596d96befb290197.json","parameterValues":[]}